### PR TITLE
Questions

### PR DESCRIPTION
**Pull request by surbhi84**

### CODE REVIEW

**Review by HeyNitin — APPROVED** [review]
Looks good to me

File: cssBundle/footer.css [hunk 1]
```diff
@@ -0,0 +1,18 @@
+/* .copyright-text {
+  padding: 0 var(--spacing-xs);
+} */
```
Comment (Aditi0505) [inline cssBundle/footer.css:3]: Can remove these if not needed.

**Review by Aditi0505 — APPROVED** [review]
looks good to me.

File: cssBundle/footer.css (hunk 1 above)
Comment (surbhi84) [reply cssBundle/footer.css:3]: okay i will

**Pull request merged**